src/main/java/Entities/Property.java

```diff
@@ -93,7 +93,7 @@ public int getHouses(){
         return this.houses;
     }
 
-    public ArrayList<String[]> loadProperties() throws FileNotFoundException {
+    public static ArrayList<String[]> loadProperties() throws FileNotFoundException {
         // return array of all properties in txt file as Strings
         // GameCreationInteractor will parse Strings to create Property instances
         ArrayList<String[]> allProperties = new ArrayList<>();
```

Comment:
Is loadProperties() only returning an empty list? Where does the property info come from?